lib/ddg-atb.js

```diff
@@ -37,13 +37,18 @@ function redirect(e) {
 }
 
 function update(){
-    var atb     = ss.storage.atb, 
-        setATB  = ss.storage.set_atb;
+    var atb = ss.storage.atb; 
 
-    if(!atb || !setATB){
+    if(!atb){
         return;
     }
 
+    if(!ss.storage.atb_set){
+        ss.storage.atb_set = true;
+    }
```

Comment:
I don't think we want to do this because it will set it to true for everyone that has the extension installed.

The existing code only sets `atb_set` to true when the user installs the extension through the duckduckgo.com site and has pulled a valid ATB param with variants from the internal code. We're doing this so that we know the exact denominator of people that installed and only receive back retention data for those users.